docs/network/ovn_kubernetes_traffic_flows.md

```diff
@@ -0,0 +1,445 @@
+# OVNKubernetes Traffic Flows in MicroShift
+
+## OVN Topology
+
+The following diagram captures the OVN network topology for the MicroShift deployment:
+
+![picture_alt](ovn_topology.svg "OVN Topology")
```

Comment:
> Nice diagram! That's exactly what I was hoping to get.
> 
> There are a couple of IPs around the "ovs join switch" that start `100.64` but I think they should probably be `10.64` based on the CIDR.

Good catch!  This is a typo, should be in the range of 100.64, I will update the diagram to use right IPs.

> 
> We have had some requests to handle network features like VLANs and bonding, and to generally provide better user control for which NICs can or cannot communicate with the cluster. Based on the diagram you provided, I wonder if we could achieve some of those goals by asking the user to create br-ex however they want, and then just having the CNI system attach to it. Would something like that work?

Yes, it will work.  We have similar use cases in openshift, where bond/vlan were created by MCO or nmstate, and br-ex is created on top of bond/vlan interfaces. In microshift, user can do:
1) manually create bond/vlan, then start microshfit (microshift-ovs-init) service
2) disable microshift-ovs-init, manually create bond/vlan and br-ex
Both shall work, user has to make sure the manual configurations persist across reboots.

Reply:
added instructions for manual vlan/bond configuration in default_cni_plugin.md doc.

Reply:
I will send another PR to cleanup the default_cni_plugin doc: remove content that is duplicate with the ovn_kubernetes_traffic_flow.md or host_networking.md.